src/parse.js

```diff
@@ -241,6 +257,22 @@ Parser.prototype.parseShortList = function(count) {
     return list;
 };
 
+// Parse a list of 24 bit unsigned integers. The length of the list can be read on the stream
+// or provided as an argument.
+Parser.prototype.parseUInt24List = function(count) {
```

Comment:
This is probably better named ``parseUInt24Array`` as a List is usually used to refer to a Linked List, but that's a minor sticking point.

Reply:
although it looks like there are other functions named this way so i don't think we should break the convention we already have.

Reply:
Absolutely, let's stick to the naming convention we already have.